apps/portal/app/components/follow/follow-modal.tsx

```diff
@@ -5,7 +5,7 @@ import { ClaimPresenter, IdentityPresenter } from '@0xintuition/api'
 
 import { multivaultAbi } from '@lib/abis/multivault'
 import { useFollowMutation } from '@lib/hooks/mutations/useFollowMutation'
-import { useCreateConfig } from '@lib/hooks/useCreateConfig'
+import { useCreateClaimConfig } from '@lib/hooks/useCreateClaimConfig'
```

Comment:
Ahh my bad! I didn't realize that this was as simple as this. I thought the hook itself was missing from the revert.